AppStore/Ukrainian/Description.iOS.txt

```diff
@@ -1,28 +1,28 @@
-Чи цікавились ви колись «хто ця актриса?», або можливо «коли вийшов цей фільм?», або «як називався той епізод мого улюбленого серіалу»? Може, ви знавець кінематографу і замислювались «скільки їм було коли вийшов цей фільм»?
+Чи цікавились ви колись «хто ця актриса?», або можливо «коли вийшов цей фільм?», або «як називався той епізод мого улюбленого серіалу»? Може, ви знавець кінематографу і замислювались «коли вийшов цей фільм, скільки років було режисеру»?
 
-Чи хотіли б ви дізнатися відповіді на усі ці питання без постійних запитів створити обліковий запис? Без реклами та без відео, що автоматично починають грати? Не покладаючись на міжнародний конгломерат, який намагається вам щось продати?
+Чи хотіли б ви дізнатися відповіді на усі ці питання без постійних запитів створити обліковий запис? Без відео, що автоматично починають грати? Не покладаючись на міжнародний конгломерат, який усюди пхає свою рекламу?
```

Comment:
Amazon is not in Ukraine, so "a multinational conglomerate that is trying to sell you things" isn't _really_ relatable

Reshuffled this paragraph so that

> Without advertisements, and without auto-playing videos? Without relying on a multinational conglomerate that is trying to sell you things?

read like

> Without auto-playing videos? Without relying on a multinational conglomerate that shoves its ads everywhere?